tests/test-class-amp-validation-utils.php

```diff
@@ -1,12 +1,12 @@
 <?php
```

Comment:
It might be good to move these tests to separate files, like `test-class-amp-validation-manager.php`.

Reply:
Yes, absolutely agree. We'll follow up with new PR to add/update the tests and also to polish the UI. 👍